docs/src/main/sphinx/object-storage/file-system-cache.md

```diff
@@ -134,6 +134,17 @@ enable and configure caching for the specific catalogs.
     smaller values are less efficient since they result in more individual downloads.
 :::
 
+## Monitoring
+
+The cache exposes the
+[Alluxio JMX client metrics](https://docs.alluxio.io/ee-da/user/stable/en/reference/Metrics-List.html#client-metrics)
+under the `org.alluxio` package, and metrics on external reads and cache reads under
+`io.trino.filesystem.alluxio.AlluxioCacheStats`. Not all JMX metrics on the workers are
```

Comment:
Maybe we should remove this as discussed.. or turn it into a positive (my vote ..)

```
Some JMX metrics are not propagated to the coordinator, and therefore not available with the [](/connector/jmx).
```

Reply:
I don't think we should document that until we understand what is the problem. Documentation is not the place to allude to potential bugs

Reply:
Okay, I've removed it.